app/src/test/java/com/safetyheads/akademiaandroida/PasswordValidatorTest.kt

```diff
@@ -0,0 +1,85 @@
+package com.safetyheads.akademiaandroida
+
+import android.content.Context
+import android.widget.EditText
+import io.mockk.mockk
+import io.mockk.verify
+import org.assertj.core.api.AssertionsForInterfaceTypes.assertThat
+import org.junit.Test
+
+
+class PasswordValidatorTest {
+
+    private val validator = PasswordValidator()
+
+    @Test
+    fun validateAttach() {
+        val editText = mockk<EditText>(relaxed = true)
```

Comment:
Please use Give-When-Then structure for each test.
https://blog.j-labs.pl/index.php?page=2017/02/Given-When-Then-pattern-in-unit-tests